packages/@ember/-internals/glimmer/lib/glimmer-tracking-docs.ts

```diff
@@ -97,6 +97,13 @@
     }
   }
   ```
+  
+  ### When `@tracked` is not needed
+  
+  * The property is already a `computed`
+  * ember-data attrs (which are computeds behind the scenes)
+  * Using Ember `get` / `set` will auto track
```

Comment:
What about passed in properties (`@something` / `this.args.something`)?

Reply:
Are those auto tracked? I don't actually know.

Reply:
Those are autotracked, yes.

Reply:
In [here](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_arguments) it describes:

>`args` and its values are automatically tracked, so there is no need to annotate them, the `fullName` getter will invalidate properly when they change and the component will rerender (if `fullName` is used in the template).